cli/cmd/version.go

```diff
@@ -35,15 +35,16 @@ var (
 	Version string
 )
 
+const keptnReleaseDocsURL = "0.8.x" // ToDo: Can we automate this?
```

Comment:
Yes: The version is input for the CLI build. 
By convention, we always use `MAJOR.MINOR.x` as docs path.

Reply:
Okay, but only if it's a semantic-version. I'll create an issue for this.

Reply:
https://github.com/keptn/keptn/issues/2863